Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleWindow.cpp

```diff
@@ -63,7 +63,8 @@ namespace ShaderManagementConsole
         msgBox.setText("Where do you want to save the list?");
         QPushButton* projectBtn = msgBox.addButton(QObject::tr("Save to project"), QMessageBox::YesRole);
         QPushButton* engineBtn = msgBox.addButton(QObject::tr("Save to engine"), QMessageBox::NoRole);
-        msgBox.exec();
+        QPushButton* cancelBtn = msgBox.addButton(QObject::tr("Cancel"), QMessageBox::RejectRole);
+        int ret = msgBox.exec();
 
         AZ::IO::Path result;
```

Comment:
declare this as an empty path but I think this should already be empty with the default constructor.

Reply:
Thanks for the tip!